test/index.test.ts

```diff
@@ -0,0 +1,10 @@
+// You can import your modules
+// import index from './index'
```

Comment:
FWIW, although this is just a comment, the proposed path is not accurate. Should be `'../lib/index'`. I updated this in PR #2.